src/docker-utils.coffee

```diff
@@ -101,27 +101,30 @@ do ->
 		Promise.using readLockImages(), ->
 			dockerProgress.pull(image, onProgress)
 
-	supervisorTag = config.supervisorImage
-	if !/:/g.test(supervisorTag)
-		# If there is no tag then mark it as latest
-		supervisorTag += ':latest'
+	normalizeRepoTag = (image) ->
```

Comment:
I mean I think it'd make sense for buildRepoTag to be in docker to also be in docker-toolbelt?